Database/FoldingCoinStoredProceduresV1.sql

```diff
@@ -248,7 +248,7 @@ BEGIN
 
 	BEGIN TRY
 		BEGIN TRANSACTION
-			SELECT @FileDownloadErrorStatusId = FoldingCoin.GetfileDownloadErrorStatusId();
+			SELECT @FileDownloadErrorStatusId = [FoldingCoin].GetFileDownloadErrorStatusId();
```

Comment:
Little confused why this wasn't found when I first made the change to case sensitive database but this was giving me errors now...maybe my test set was different?